pyblish_nuke/nuke_path/menu.py

```diff
@@ -2,11 +2,8 @@
 
 import nuke
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
Why are we getting rid of the logging here? Does logging work without this?

As far as I remember we needed this, for Nuke to output logging to Nukes script editor.